data/src/main/java/me/androidbox/data/remote/event/VerifyVisitorEmailUseCaseImp.kt

```diff
@@ -0,0 +1,36 @@
+package me.androidbox.data.remote.event
+
+import me.androidbox.data.mapper.toAttendee
+import me.androidbox.data.remote.model.response.AttendeeEmailVerifyResponseDto
+import me.androidbox.data.remote.network.event.EventService
+import me.androidbox.data.remote.util.CheckResult.checkResult
+import me.androidbox.domain.authentication.ResponseState
+import me.androidbox.domain.authentication.model.Attendee
+import me.androidbox.domain.event.usecase.VerifyVisitorEmailUseCase
+import javax.inject.Inject
+
+class VerifyVisitorEmailUseCaseImp @Inject constructor(
```

Comment:
The data layer is neither the place for use cases nor are use cases the place to put data-related calls. Use cases should be completely isolated from 3rd party SDKs or the Android SDK and just contain business logic or work with abstractions from the domain layer (such as your repo interface).
Also, there is no point in abstracting a use case since you'll never have a second implementation of it.